packages/manager/apps/container/src/payment-modal/styles.scss

```diff
@@ -0,0 +1,20 @@
+@import 'bootstrap4/scss/bootstrap';
+@import '@ovh-ux/manager-hub/src/variables.scss';
```

Comment:
importing things from src seems like a bad idea

Reply:
this import is commonly used in the container. By the way, as I switched to ODS I've removed it